silica/compile.py

```diff
@@ -10,7 +10,7 @@
 from silica.coroutine import Coroutine
 from silica.cfg import ControlFlowGraph, BasicBlock, HeadBlock
 from silica.cfg.control_flow_graph import render_paths_between_yields, build_state_info, render_fsm, get_constant
-from silica.ast_utils import get_ast
+from silica.ast_utils import *
```

Comment:
Minor style comment, I know this is a tedious change, but I prefer if we use `module.func` rather than `import *`. Mainly, this interacts better with tooling for things like autocompletion, also makes it more explicit where things are coming from in the code.

We could do
```
import silica.ast_utils as ast_utils
```

then prepend each function call with `ast_utils`, e.g. `ast_utils.is_name(...)`.

We could also use some shorthand to make it easier to type, maybe `import silica.ast_utils as au`? `au` is not super clear through